data/tutorials/gs_00_up_and_running.md

```diff
@@ -44,6 +44,28 @@ Find all the installation instructions for both Unix-like systems and Windows in
 * Linux or macOS: [Installation for Unix, including Linux and macOS](#installation-for-unix)
 * Windows: [Installation for Windows](#installation-for-windows)
 
+### Installation using Ocaml Platform Installer
+
+As of 2023, the OCaml Plarform Installer, presented here, is new recommended way
+to install OCaml in Unix. The `ocaml-platform` binary, automates and set up of a
+OCaml development environment. However, you need first to install the few system
+dependencies of the OCaml environment, such as a C compiler (e.g. `gcc`) and
+other system tools: `bzip2`, `make`, `bubblewrap`, `patch`, `curl` and `unzip`.
+In most architecture, you can install them using your package manager, for
+example on Ubuntu or Debian:
+<!-- $MDX skip -->
+```bash
+$ sudo apt install build-essential bubblewrap unzip
+```
+
+In MacOS, having installed [Xcode](https://developer.apple.com/xcode) is the only requirement.
+
+You can now download and run the installer script and then call ocaml-platform:
+```shell
+$ bash < <(curl -sL https://ocaml.org/platform/installer.sh)
+$ ocaml-platform
```

Comment:
Nice!

If this is merged, maybe we should also modify the installer's README to use the same link?
On one hand, it is a nice url, on the other hand, it is less transparent for users that they are downloading the script included in the latest github release.

Reply:
Good point. That makes me want to write a patch to `curl` to have an option (say `-3`) to only display redirects. We'd have `bash < <(curl -3L https://ocaml.org/platform/installer.sh)` in the docs and when executed by the users they would see something like that:
```
2022-12-08 15:30:26 https://ocaml.org/platform/installer.sh [redirect]
2022-12-08 15:30:26 https://github.com/tarides/ocaml-platform-installer/releases/latest/download/installer.sh [redirect]
2022-12-08 15:30:27 https://github.com/tarides/ocaml-platform-installer/releases/download/0.6.0/installer.sh
```

|